lib/lighthouse/matchers.rb

```diff
@@ -27,6 +28,13 @@ def runner
         @runner ||= proc { |cmd| `#{cmd}` }
       end
 
+      def chrome_flags
+        return unless @chrome_flags
```

Comment:
Mainly for learning: 

What does this catch that `is_a?(Array)` won't?

Or is it just about strictly enforcing the return type to `Array | nil`?

Reply:
We want to return `nil` if there is no value (as in it is falsey). This allows us to not add the option at all if the option is not specified.

Reply:
Right, so it's the second one. thanks for confirming 👍